### PR TITLE
Branch tutorial

### PR DESCRIPTION
**Pull request by audhalbritter**
Updated text in the tutorial. Still need to add proper pictures.

### CODE REVIEW

File: DataManagement/BranchTutorial.Rmd [hunk 1]
```diff
@@ -85,87 +85,100 @@ knitr::include_graphics("Pics/6. add_url.png")
 <br/><br/>
 <br/><br/>
 
-## Branch
+## What is a branch?
 
 You have forked a repository and are ready to start working on the project. Let's talk about branches.
 
-A branch lets you develop code or test an idea without affecting the original project. Once you are happy with the new code, or the idea is tested and working you can merge the branch with master branch. If the idea did not work or you have another idea, the branch can be deleted. None of this will affect the original project.
+A branch lets you develop code or test an idea without affecting the original project. This means you can bravely develop code that works or does not work. Once you are happy with the new code, or the idea is tested and working you can merge the branch you are working on with master branch. If the idea did not work or you have another idea, the branch can be deleted and nobody ever needs to know about it. Make a new branch and start all over again. None of this will affect the original project.
```
Comment (richardjtelford) [inline DataManagement/BranchTutorial.Rmd:92]: Top tip - write one sentence per line.
New sentence, new line. 

Should we talk about merging with the master branch here? On your own project this is really useful, but if you are working on a fork, you really shouldn't merge. Not sure how best to handle this.

File: DataManagement/BranchTutorial.Rmd [hunk 2]
```diff
@@ -85,87 +85,100 @@ knitr::include_graphics("Pics/6. add_url.png")
...
+Add a comment about your pull request if necessary and create the pull request.
+<br/><br/>
 <br/><br/>
 ```{r pull-request-2}
 knitr::include_graphics("Pics/13. Pullrequest 2.png")
 ```
 <br/><br/>
-Once you have created your pull request, github will check if there are any merge conflict.
+<br/><br/>
+Once you have created your pull request, github will check if there are any merge conflict. If no merge conflicts occur, the box will become green. If there are merge conflicts, click on the box resolve merge conflicts and fix the problems.
```
Comment (richardjtelford) [inline DataManagement/BranchTutorial.Rmd:152]: GitHub

Comment (richardjtelford) [inline DataManagement/BranchTutorial.Rmd:152]: Somewhere we will have to explain what merge conflicts are and how to fix them

File: DataManagement/BranchTutorial.Rmd [hunk 3]
```diff
@@ -118,57 +134,63 @@ write code, work in small logical steps (e.g. write a function, fix a function,
...
 <br/><br/>
 Now it's time for you to relax, have some cake 🍰, you have earned it!
 <br/><br/>
 
-### Master accepts pull request (or not)
+### Owner accepts pull request (or not)
 The owner of the original project has to evaluate your changes to the project. The owner can accept your code, decline or make suggestions for improvement.
 
 You will get a notification if your pull request has been accepted.
 <br/><br/>
```
Comment (richardjtelford) [inline DataManagement/BranchTutorial.Rmd:169]: Celebration gif

File: DataManagement/BranchTutorial.Rmd (hunk 2 above)
Comment (audhalbritter) [reply DataManagement/BranchTutorial.Rmd:152]: I know...

**Pull request merged**